### PR TITLE
Wave strength controller

### PR DESCRIPTION
**Pull request by NjuuOwO**

### CODE REVIEW

File: src/views/world/view_world.cpp [hunk 1]
```diff
@@ -137,6 +137,17 @@ namespace big
...
 				deleting = false;
 			});
+
+			ImGui::SeparatorText("Miscellaneous");
+
+			int override_waves_as_int = static_cast<int>(g.world.override_waves);
+			if (ImGui::SliderInt("Waves strength", &override_waves_as_int, 0, 5))
+			{
+				g.world.override_waves = static_cast<float>(override_waves_as_int);
+				MISC::WATER_OVERRIDE_SET_STRENGTH(g.world.override_waves);
```
Comment (Yimura) [inline src/views/world/view_world.cpp:147]: Executing a native in the UI thread, this should be done in a fiber.

**Review by Yimura — CHANGES_REQUESTED** [review]
Also this could be made into a Lua script easily.

File: src/views/world/view_world.cpp (hunk 1 above)
Comment (NjuuOwO) [reply src/views/world/view_world.cpp:147]: I don't know what you mean

Comment (gir489returns) [reply src/views/world/view_world.cpp:147]: > I don't know what you mean

Which is exactly why this PR is gonna be rejected and you need to make this a Lua instead.

**Comment by NjuuOwO** [comment]
ye he's right

**Pull request closed without merging**